app/assets/stylesheets/components/common/_contact_panel.scss

```diff
@@ -3,25 +3,43 @@
 // Styleguide Contact Panel
 
 .contact-panel {
+  // N.B. This is to over-ride styles from Yeast
+  // Should be added to that for a final deployment
+  @include column(12);
+
   color: $color-grey-medium-dark;
   text-align: center;
-  border-bottom: 1px solid $color-newsletter-input-border;
   padding: $baseline-unit*2 0;
 
   @include respond-to($mq-s) {
     padding: $baseline-unit*4 0;
-    margin-bottom: $baseline-unit*2;
+  }
+
+  @include respond-to($mq-m) {
+    border-bottom: 1px solid $color-newsletter-input-border;
   }
 
   @include respond-to($mq-l) {
     border-bottom: 0;
+    // N.B. This is to over-ride styles from Yeast
+    // Should be added to that for a final deployment
+    width: 100%;
+    margin: 0;
   }
```

Comment:
As discussed with Jason, these styles (width, margin) are OK to leave in frontend - but can we delete the comments?